esp32_ulp/opcodes.py

```diff
@@ -267,6 +268,22 @@ def make_ins(layout):
 ARG = namedtuple('ARG', ('type', 'value', 'raw'))
 
 
+def eval_arg(arg):
+    parts = []
+    for token in split_tokens(arg):
+        if symbols.has_sym(token):
+            _, _, sym_addr = symbols.get_sym(token)
```

Comment:
sym_value is maybe more suitable as name here.

Reply:
Will update.